cmd/monaco/cmdutils/cmdutils_test.go

```diff
@@ -75,7 +75,7 @@ func TestVerifyClusterGen(t *testing.T) {
 	}
 	for _, tt := range tests {
 		t.Run(tt.name, func(t *testing.T) {
-			if err := VerifyClusterGen(tt.args.envs); (err != nil) != tt.wantErr {
+			if err := VerifyClusterGen(tt.args.envs); err == tt.wantErr {
```

Comment:
nit: it's not really an Error anymore. Maybe name this ok like you did below and changing the test cases to always define the wanted true/false result?